Directory.Build.targets

```diff
@@ -10,20 +10,6 @@
     <PublishTrimmed Condition=" '$(TargetFramework)' == 'net6.0-ios' or '$(TargetFramework)' == 'net6.0-maccatalyst' ">false</PublishTrimmed>
```

Comment:
@mattleibow can we remove this now?

Reply:
Maybe. I know I did in my pr and I think the issue was that we were using Xcode 12. Now with Xcode 13 all over, we probably can remove it.